router/appRest.js

```diff
@@ -503,12 +504,20 @@ function sendResponseWhatsapp(response,responseBody, recipient, textContent) {
 		},
 		json: true
 	};
-	request(options, function (error, response, body) {
-		// adding logs for debugging purposes only
-		LOG.info("Sending response to whatsapp is failing :", error);
-		LOG.info("Sending response to whatsapp is failing :", response);
-		LOG.info("Sending response to whatsapp is failing :", body);
-		if (error) throw new Error(error);
+	request(options, function (error, resp, body) {
+		const edata = {
+			type: "system",
+			level: "INFO",
+			requestid: response.req.headers["x-request-id"] ? response.req.headers["x-request-id"] : "",
+			message: body,
```

Comment:
Print only 500 characters max

Reply:
Fixed: only logged message from body, and logged complete body in logger.